makefile

```diff
@@ -430,7 +430,10 @@ endif
 	@cp ${ECMD_ROOT}/ecmd-core/ext/cip/cmd/help/cipinstruct.htxt ${INSTALL_PATH}/help/.
 	@cp ${ECMD_ROOT}/ecmd-core/ext/cip/cmd/help/cipgetmemproc.htxt ${INSTALL_PATH}/help/.
 	@cp ${ECMD_ROOT}/ecmd-core/ext/cip/cmd/help/cipputmemproc.htxt ${INSTALL_PATH}/help/.
-	@cp ${ECMD_ROOT}/out_${TARGET_ARCH}/bin/ecmd.htxt ${INSTALL_PATH}/help/.
+	@cp ${ECMD_ROOT}/ecmd-core/cmd/help/ecmd.htxt ${INSTALL_PATH}/help/.
```

Comment:
Why is this path changed?  ecmd.htxt does not exist in that path?  It exists in the out_ dir because it is generated as part of the build.

Reply:
I was trying make install today and it was failing. so , it is not created under that path. I haven't investigated .. i can investigate and drop this change for now?

Reply:
It counts on the ecmd subrepo itself having been built - it would fail without that.

Reply:
ecmd subrepo is getting build. I went to ecmd/ecmd-core/cmd/ 

did "make ecmd.htxt" and it created and copied the file to help path. Otherwise, this autogeneration isn't happening.

Reply:
I will debug this separately for now and send a PR